pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -44,11 +45,16 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 	platform := ic.Config.Platform.Name()
 	switch platform {
 	case aws.Name:
+		var preexistingNetworking bool
+		if len(ic.Config.AWS.Subnets) > 0 {
+			preexistingNetworking = true
+		}
```

Comment:
nit (or minor suggestion you are free to ignore): 
one liner: `preexistingNetwork := len(ic.Config.AWS.Subnets > 0`